### DISCUSSION TITLE
How to do Windows SDDC (Software-Defined Sata Center) certification

### DISCUSSION THREAD

**Discussion by springeee** [community/community#129825] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all:

I want to do nvme ssd's windows SDDC certification .
![image](https://github.com/community/community/assets/72604063/4add4ed2-6150-454f-967b-00692dcabe07)
the follow all test will be run or only run S2D will be ok ？
![image](https://github.com/community/community/assets/72604063/881624eb-4e7d-4bac-8c91-3dbb9f148019)

Thanks,
Regards
Springeee

**Comment by ebndev** [comment]
Hi @springeee, thanks for being a part of the GitHub Community!

I saw that you also posted [this](https://learn.microsoft.com/en-us/answers/questions/1753304/how-to-do-windows-sddc-software-defined-sata-cent) in Microsoft's forum, so I will be closing this Discussion as the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
